core/integration/dcs/plugin.go

```diff
@@ -1416,17 +1415,6 @@ func (p *Plugin) CallStack(data interface{}) (stack map[string]interface{}) {
 					WithField("run", runNumber64).
 					Debug(logMsg)
 
-				the.EventWriterWithTopic(TOPIC).WriteEvent(&pb.Ev_IntegratedServiceEvent{
-					Name:                call.GetName(),
-					OperationName:       call.Func,
-					OperationStatus:     pb.OpStatus_ONGOING,
-					OperationStep:       "perform DCS call: StartOfRun",
-					OperationStepStatus: pb.OpStatus_DONE_ERROR,
-					EnvironmentId:       envId,
-					Payload:             string(payloadJson[:]),
-					Error:               logMsg,
-				})
-
```

Comment:
I agree with this change. Beyond the question of whether the stream `io.EOF` was timely or not from the DCS side, it could be argued that a stream being closed either way should not generate a Kafka event, because it concerns a technicality of DCS-ECS communication and says nothing about the actual state of the DCS.

However, the behaviour should be consistent across all DCS operations. @knopers8 could you please make sure this change also applies to PFR and EOR?

Reply:
sure, I will do so.